src/mobility/utils.ts

```diff
@@ -101,8 +102,10 @@ export const needsReload = (
     shownArea.range,
   );
 
+  if (!prevAreaFeature || !newAreaFeature) return true;
```

Comment:
Can these be `undefined`?

Reply:
The new function does not allow undefined as before, instead requires a valid object, but those could be undefined yes.

Reply:
By looking at the code I can't see how they can be `undefined`? Seems like `extend` can't return `undefined`, and if it does then the method signature is wrong.

Reply:
This is what the lint returns after remove the condition:

```
src/mobility/utils.ts:106:46 - error TS2322: Type 'Feature<Polygon | MultiPolygon, GeoJsonProperties> | undefined' is not assignable to type 'Feature<Polygon | MultiPolygon, GeoJsonProperties>'.
  Type 'undefined' is not assignable to type 'Feature<Polygon | MultiPolygon, GeoJsonProperties>'.

106   const diff = difference(featureCollection([newAreaFeature, prevAreaFeature]));
                                                 ~~~~~~~~~~~~~~

src/mobility/utils.ts:106:62 - error TS2322: Type 'Feature<Polygon | MultiPolygon, GeoJsonProperties> | undefined' is not assignable to type 'Feature<Polygon | MultiPolygon, GeoJsonProperties>'.
  Type 'undefined' is not assignable to type 'Feature<Polygon | MultiPolygon, GeoJsonProperties>'.

106   const diff = difference(featureCollection([newAreaFeature, prevAreaFeature]));
                                                                 ~~~~~~~~~~~~~~~


Found 2 errors in the same file, starting at: src/mobility/utils.ts:106
```

Reply:
Aha, I see the signature from `@turf/buffer` was changed in latest version!